### PR TITLE
benchmarks in bench_templates.py refactored

### PR DESCRIPTION
**Pull request by deepakdinesh1123**
benchmarks refactored
- template_compile moved to template_benchmarks/template_compilation
- template_render and template_render_simple moved to template_benchmarks/template_render
- templates moved to template_benchmarks/template_render/templates

### CODE REVIEW

**Comment by smithdc1** [comment]
I've rebased this one as well, if you could review that would be much appreciated. Thank you!

File: benchmarks/template_benchmarks/template_compilation/benchmark.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+SCRIPT_CONTENT_URL = "/some/prefix"
+WEBSITE_DOMAIN = "http://www.somedomain.com"
+SHOW_ALT_HEADER = "True"
+
+
+class TemplateCompile:
+    def setup(self):
+        bench_setup()
+        self.context = {
+            "objects1": objects1,
```
Comment (smithdc1) [inline benchmarks/template_benchmarks/template_compilation/benchmark.py]:
```suggestion
            "objects1": [object(), object(), object(), object(), object()],
```

We're not reusing these objects so I think we can create them as part of the setup.

**Review by smithdc1 — COMMENTED** [review]
Thanks for this, just one comment inline then one query about the templates that are now deleted `lte_13` etc. Django 1.3 came out more than a decade ago and so I agree it makes sense to drop these now. 

Can you remind me which benchmarks were using those. I'm interested to understand if they have been recfatored/updated them as needed?

File: benchmarks/template_benchmarks/template_compilation/benchmark.py (hunk 1 above)
Comment (deepakdinesh1123) [reply benchmarks/template_benchmarks/template_compilation/benchmark.py]: I have made the changes

**Comment by deepakdinesh1123** [comment]
I checked, but I could not find any benchmarks that were using `lte_13` so I had removed it.

**Comment by smithdc1** [comment]
> I checked, but I could not find any benchmarks that were using `lte_13` so I had removed it.

Thanks. Let's merge this. 🎉

**Pull request merged**